sound/soc/sof/intel/hda.c

```diff
@@ -1176,8 +1177,15 @@ int hda_dsp_remove(struct snd_sof_dev *sdev)
 				SOF_HDA_INT_CTRL_EN | SOF_HDA_INT_GLOBAL_EN, 0);
 
 	/* disable cores */
-	if (chip)
-		hda_dsp_core_reset_power_down(sdev, chip->host_managed_cores_mask);
+	if (chip && chip->power_down_dsp) {
+		ret = chip->power_down_dsp(sdev);
+		if (ret) {
+			dev_err(sdev->dev, "error: power_down_dsp op returns %d\n", ret);
```

Comment:
please remove the error: prefix, we no longer use it.

Reply:
I did rapid fire in 5 min. I made the update and slightly changed. Can you check?